internal/zinc-scripted/src/test/scala/sbt/internal/inc/ZincFileCommands.scala

```diff
@@ -1,18 +1,80 @@
 package sbt.internal.inc
 
 import java.io.File
+import java.nio.file.Paths
 
 import sbt.internal.scripted.FileCommands
+import sbt.io.IO
 
 class ZincFileCommands(baseDirectory: File) extends FileCommands(baseDirectory) {
   override def commandMap: Map[String, List[String] => Unit] = {
     super.commandMap + {
       "pause" noArg {
         // Redefine pause not to use `System.console`, which is too restrictive
-        println(s"Pausing in $baseDirectory. Press enter to continue.")
+        println(s"Pausing in $baseDirectory Press enter to continue.")
```

Comment:
Why remove the dot?

Reply:
This is annoying dot :) If you want to copy the path with double click, it copies the dot and you need to remove it to access directory. Though I could agree that it would be nice to have it nice in English as well but it was just so annoying. Maybe it is also annoying someone else so I included it in PR. Though I can revert and remove dot locally whenever it annoys me.

Reply:
I will at least put it on separate commit